app/src/main/java/romina/valiunas/weatherapp/utils/LongToDateConverter.kt

```diff
@@ -7,7 +7,9 @@ class LongToDateConverter {
 
     fun getTheDay(longDate: Long): String {
```

Comment:
Define Utils class methods inside a companion object to avoid instantiation. Also, remove wildcard imports.

Reply:
companion object created